src/spotify_dl.py

```diff
@@ -58,9 +58,16 @@ class SpotifySong:
     url = f"https://open.spotify.com/track/{id}"
 
 
-def _get_track_local_title(title: str, artist: str) -> str:
-    return f"{title} - {artist}"
+# def _get_track_local_title(title: str, artist: str) -> str:
+#     return f"{title} - {artist}"
 
+def assemble_track_custom_title(title: str, artist: str = "", id: int = 1,
```

Comment:
can you change "id" to "ind" or "track_num" for clarity? the track's actual ID is a long hash from the API

Reply:
> can you change "id" to "ind" or "track_num" for clarity? the track's actual ID is a long hash from the API

Yeah sure, changed to track_num. Want me to change the template itself to "{track_num}". I think it would be a better fit than 'id' too.